src/node/rpc/test/frontend_test.cpp

```diff
@@ -86,9 +88,23 @@ class TestNoCertsFrontend : public ccf::RpcFrontend
   }
 };
 
+class TestForwardingFrontEnd : public ccf::RpcFrontend
+{
+public:
+  TestForwardingFrontEnd(Store& tables) : RpcFrontend(tables)
+  {
+    auto empty_function = [this](RequestArgs& args) {
+      return jsonrpc::success();
+    };
+    // Note that this a Write function
```

Comment:
It would be preferable to explain why, rather than state that it's a Write function, which is apparent from the parameters.